README.md

```diff
@@ -64,17 +64,19 @@ Behaviour can be configured in the `app.json` under the `svgAppIcon` field. For
   "displayName": "example",
   "svgAppIcon": {
     "foregroundPath": "./icon/icon-foreground.svg",
-    "backgroundPath": "./icon/icon-background.svg"
+    "backgroundPath": "./icon/icon-background.svg",
+    "excludePlatforms": ["android"]
```

Comment:
The original issue suggested a `platforms` configuration, without the negation. Seems like you decided to change it in the implementation. Was there some specific reasoning behind this?

Personally, I feel that avoiding the negation, and just listing the platforms for which icons should be generated, would be easier to understand. This of course would mean that any new platform would need to be added separately, but that feels unlikely to happen very often.

Reply:
I was under the false impression, that the `platforms` array would be merged with `defaultCliConfig` and cause issues. Since this doesn't seem to be the case we could remove the negation if you aren't planning on adding more platforms anyways. Otherwise I think the `excludePlatforms` approach is more future proof and can avoid issues when adding new platforms, because all exclusions would be explicitly set by the user.

Reply:
Also using the `exclude` negation in the internal config allowed me to easily make the parameter optional so as not to break existing tests. Removing the negation would make it a mandatory parameter (since it can't just be `undefined` and falsy anymore) or add the need for additional checks.

Reply:
If you don't have a strong preference for `excludePlatforms`, I would suggest we use `platforms`. It is similar to the configuration react native cli provides (https://www.github.com/react-native-community/cli/tree/master/docs%2Fcommands.md), and what expo provides (https://docs.expo.io/versions/latest/config/app/).

In the future,we might even want to respect the top level `app.json` expo config `platforms` value.